crates/polars-plan/src/dsl/function_expr/rolling.rs

```diff
@@ -130,39 +126,39 @@ fn convert<'a>(
 }
 
 pub(super) fn rolling_min(s: &Series, options: RollingOptions) -> PolarsResult<Series> {
-    s.rolling_min(options.clone().try_into()?)
+    s.rolling_min(options.clone().into())
```

Comment:
Why do we clone here? We have town the `options`.

Reply:
no reason as far as I can tell, I've removed it